datasets/greek_legal_code/README.md

```diff
@@ -0,0 +1,195 @@
+
+Greek_Legal_Code (GLC) is a dataset consisting of approx. 47k legal resources from Greek legislation. The origin of GLC is “Permanent Greek Legislation Code - Raptarchis”, a collection of Greek legislative documents classified into multi-level (from broader to more specialized) categories.
+
+**Topics**
+
+GLC consists of 47 legislative volumes and each volume corresponds to a main thematic topic. Each volume is divided into thematic sub categories which are called chapters and subsequently, each chapter breaks down to subjects which contain the legal resources. The total number of chapters is 389 while the total number of subjects is 2285, creating an interlinked thematic hierarchy. So, for the upper thematic level (volume) GLC has 47 classes. For the next thematic level (chapter) GLC offers 389 classes and for the inner and last thematic level (subject), GLC has 2285 classes.
+
+GLC classes are divided into three categories for each thematic level: frequent classes, which occur in more than 10 training documents and can be found in all three subsets (training, development and test); few-shot classes which appear in 1 to 10 training documents and also appear in the documents of the development and test sets, and zero-shot classes which appear in the development and/or test, but not in the training documents.
+
+
+### Supported Tasks and Leaderboards
+
+The dataset supports:
+
+**Multi-class Text Classification:** Given the text of a document, a model predicts the corresponding class.
+
+**Few-shot and Zero-shot learning:** As already noted, the classes can be divided into three groups: frequent, few-shot, and zero- shot, depending on whether they were assigned to more than 10, fewer than 10 but at least one, or no training documents, respectively.
+
+| Level | Total | Frequent | Few-Shot (<10) | Zero-Shot |
+|---|---|---|---|---|
+|Volume|47|47|0|0|
+|Chapter|389|333|53|3|
+|Subject|2285|712|1431|142|
+
+### Languages
+
+All documents are written in Greek.
+
+## Dataset Structure
+
+### Data Instances
+
+
+```json
+{
+  "text": "179. ΑΠΟΦΑΣΗ ΥΠΟΥΡΓΟΥ ΜΕΤΑΦΟΡΩΝ ΚΑΙ ΕΠΙΚΟΙΝΩΝΙΩΝ Αριθ. Β-οικ. 68425/4765 της 2/17 Νοεμ. 2000 (ΦΕΚ Β΄ 1404) Τροποποίηση της 42000/2030/81 κοιν. απόφασης του Υπουργού Συγκοινωνιών «Κωδικοποίηση και συμπλήρωση καν. Αποφάσεων» που εκδόθηκαν κατ’ εξουσιοδότηση του Ν.Δ. 102/73 «περί οργανώσεως των δια λεωφορείων αυτοκινήτων εκτελουμένων επιβατικών συγκοινωνιών». ",
+  "volume": "ΣΥΓΚΟΙΝΩΝΙΕΣ",
+  "chapter": "ΛΕΩΦΟΡΕΙΑ",
+  "subject": "ΚΟΙΝΑ ΤΑΜΕΙΑ ΕΚΜΕΤΑΛΛΕΥΣΕΩΣ ΛΕΩΦΟΡΕΙΩΝ (Κ.Τ.Ε.Λ.)"
+}
+```
+
+### Data Fields
+
+The following data fields are provided for documents (`train`, `dev`, `test`):
+
+`text`: (**str**)  The full content of each document, which is represented by its `header` and `articles` (i.e., the `main_body`).\
+`volume`: (**str**)  The volume-level class it belongs to.\
+`chapter`: (**str**)  The chapter-level class it belongs to.\
+`subject`: (**str**)  The subject-level class it belongs to.
```

Comment:
maybe we can list the name of the classes here ? Or at least for the "volume" case ?